testconf/imqsauth.json

```diff
@@ -0,0 +1,18 @@
+{
+	"Authaus": {
+		"Log": {
+			"Filename": ".\\log\\imqsauth.log"
```

Comment:
Just a humble request... Could you test `/`? I am pretty sure go automatically fixes the path separator. Which should mean that this should work for linux as well, but a single slash looks a tiny bit better than two.

Reply:
Done